### DISCUSSION TITLE
Wait, is Sketchware pro getting a UI redesign?

### DISCUSSION THREAD

**Discussion by RoyTheReal** [Sketchware-Pro/Sketchware-Pro#964] [General]
I just downloaded, feat: Updated the entire app font #762. The new UI looks AMAZING. Just needs a few optimizations and tweaks. Is this coming to the next update? Looks AMAZING.

**Comment by ilyassesalama** [comment]
Yes, you can always check our progress in [Material Redesign Branch](https://github.com/Sketchware-Pro/Sketchware-Pro/tree/material-redesign). We push commits regularly.
We just started 2 days ago. Of course you'll see issues with it.

**Reply by RoyTheReal** [reply]
Awesome, good job sir. Just please make more optimizations and make the design more consistent. I'm not a huge fan of the super white color when working on an app but keep up the good work.
